mods/ra/maps/survival01/survival01.lua

```diff
@@ -66,45 +66,38 @@ SovietBuildings = { Barrack1, SubPen, RadarDome, AdvancedPowerPlant1, AdvancedPo
 IdleTrigger = function(units, dest)
 	Utils.Do(units, function(unit)
 
-		Trigger.OnIdle(unit, function()
-			local bool = Utils.All(units, function(unit) return unit.IsIdle end)
-			if bool then
-				Utils.Do(units, function(unit)
-					if not unit.IsDead then
-						Trigger.ClearAll(unit)
-						Trigger.AfterDelay(0, function()
-							if not unit.IsDead then
-								if dest then unit.AttackMove(dest, 3) end
-								Trigger.OnIdle(unit, unit.Hunt)
-								Trigger.OnCapture(unit, function()
-									Trigger.ClearAll(unit)
-								end)
-							end
-						end)
-					end
-				end)
-			end
-		end)
+		if not unit.IsDead then
+			Trigger.OnIdle(unit, function()
+				local bool = Utils.All(units, function(unit) return unit.IsIdle end)
```

Comment:
It only sets it up when all other units are `Idle` as well. I already thought about introducing an `Trigger.OnAllIdle` as followup.

However, even if this is bogus: This is not part of the PR, I just moved already existing code that works fine.